src/encoded/tests/test_create_mapping.py

```diff
@@ -16,7 +16,7 @@
 pytestmark = [pytest.mark.setone, pytest.mark.working]
 
 
-@pytest.mark.parametrize('item_type', ITEM_INDEX_ORDER)
+@pytest.mark.parametrize('item_type', ITEM_INDEX_ORDER())
```

Comment:
This relates to my comment in other repos about turning this into a function. I really don't like this and would like to see either 
```
ITEM_INDEX_ORDER = ...call to other-named function that computes it...
```
or a direct call to that other-named function here.

The reason for all-uppercase is to say "this is constant" and the reason for `()` is to say "this is variable and must be computed", so this conflicted/confusing naming is what is stressing me as much as the change in semantics.

Reply:
I can change the name to not be upper-case, e.g. get_item_index_order(), but I needs to be a function because it is now coming in via the app_project mechanism (i.e. app_project().loadxl_order(), i.e. can change per downstream repo - smaht-portal, cgap-portal, fourfront).

Reply:
Name change (to loadxl_order) made to this function.